app/packs/src/stores/alt/stores/ElementStore.js

```diff
@@ -1226,14 +1251,22 @@ class ElementStore {
   }
 
   synchronizeElements(previous) {
-    const { selecteds } = this.state;
+    const { selecteds, refreshCoefficient } = this.state;
```

Comment:
It's not your naming, but what's `selecteds`? Selected samples?

Reply:
selecteds is made to update the state of the opened tabs, if I still remember correctly